sdk/src/ingredient.rs

```diff
@@ -407,8 +406,6 @@ impl Ingredient {
         if let Some(opt_title) = options.title(path) {
             ingredient.title = opt_title;
         }
-        // read the file into a buffer for processing
-        let buf = std::fs::read(path).map_err(wrap_io_err)?;
```

Comment:
I used to use the same buffer for hash and load_from_memory, so this all works out fine now.

Reply:
The problem was when I loaded a 40GB file this buf had trouble on some systems.  We no longer do that.

Reply:
yes, I'm just saying this is fine now since I didn't need the buffer anymore anyway.

Reply:
Got, I wasn't sure if you wanted to keep it.